### PR TITLE
adding basic instructions for the --export feature

### PR DESCRIPTION
**Pull request by slateem**
I would like to propose adding some additional basic instructions for using the "--export" feature on the documents page (https://docs.openbb.co/terminal/guides/advanced/data) to help new users. (I am a new contributor so sorry in advance for any mistake in this process).

# Description

- [ ] Adding basic instructions for using --export feature
- [ ] N/A
- [ ] N/A
- [ ] To provide instructions to help new users
- [ ] N/A

# How has this been tested?

N/A, just adding some content

* Please describe the tests that you ran to verify your changes.
* Provide instructions so we can reproduce.
* Please also list any relevant details for your test configuration.
- [ ] Make sure affected commands still run in terminal
- [ ] Ensure the SDK still works
- [ ] Check any related reports

# Checklist:

- [ ] Update [our Hugo documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/website).
- [ ] Update our tests following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/tests).
- [ ] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).
- [ ] If a feature was added make sure to add it to the corresponding [scripts file](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/scripts).

# Others
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

**Comment by deeleeramone** [comment]
@slateem, great idea! I hope you don't mind, I made some changes; feel free to adapt to make it your own!

File: website/content/terminal/guides/advanced/data.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ This folder contains all things user-created. For example:
 
 The location of this folder can be set by the user from the `/settings` menu. There should be no need to update paths in this menu unless the folders have been moved manually.
 
-<img width="1117" alt="The settings menu" src="https://user-images.githubusercontent.com/85772166/195736718-a1b821da-5977-437a-bd18-b44add2a29a2.png"></img>
+[The Settings Menu](https://user-images.githubusercontent.com/85772166/195736718-a1b821da-5977-437a-bd18-b44add2a29a2.png "The Settings Menu")
```
Comment (JerBouma) [inline website/content/terminal/guides/advanced/data.md:27]: Please make this an image, it is now a link. It should contain a `!` in front.

**Review by JerBouma — CHANGES_REQUESTED** [review]
Minor detail, other than that looks good.

**Comment by piiq** [comment]
new contributors are ❤️ @slateem

**Comment by slateem** [comment]
So here is what happened. In commit id bf74ff3 I forgot to save my changes before pushing the file, and so it only pushed the .vs/slnx.sqlite file (I guess vscode added that when I cloned). 

Then in 31f95f5, I pushed the correct (I think) requested changes. 

So now im not sure what to do about bf74ff3. Sorry guys...I'm still learning git.

**Comment by piiq** [comment]
@slateem Delete the `.vs/...` file locally and commit the deletion. After that the diff of the PR should not include this binary

**Review by piiq — APPROVED** [review]

**Comment by slateem** [comment]
Thanks for the help all! Love what you guys are doing!

**Pull request merged**